packages/x-charts-pro/src/context/ZoomProvider/useSetupPan.ts

```diff
@@ -77,7 +79,13 @@ export const useSetupPan = () => {
       }
       isDraggingRef.current = true;
       setIsInteracting(true);
-      touchStartRef.current = event.clientX;
+
+      const point = getSVGPoint(element, event);
+      touchStartRef.current = {
+        x: point.x,
+        minX: zoomRange[0],
+        maxX: zoomRange[1],
+      };
```

Comment:
I proposed to save the initial state rather than updating it each time the mouse moves.

Otherwise you get a lot of small steps which would require to be better handle decimal precision